modules/kubernetes-addons/opentelemetry-operator/main.tf

```diff
@@ -1,17 +1,33 @@
-module "operator" {
-  source        = "../helm-addon"
-  helm_config   = local.helm_config
-  irsa_config   = null
+module "cert_manager" {
```

Comment:
this may be related to my question above about removing cert_manager, can you share more info about this change? what we did before and what we propose to do now.

Reply:
Fair point -- I included cert manager inside the opentelemetry-operator module because it is an absolute pre-condition for the add-on. That being said, I have tried a dependency on the kubernetes operator main file, however, I encountered a terraform blocker where `for_each` (which is used to split permissions manifests) and `count` are used. 
If you have another suggetion, I'll take it